src/get-started/install/_windows-desktop-setup.md

```diff
@@ -1,20 +1,18 @@
 ## Windows setup
 
 {{site.alert.warning}}
-  **Beta (Win32) and Dev (UWP)!**
+  **Beta (Win32) and Master (UWP)!**
```

Comment:
sorry for the delay -- I think we should call this Alpha, even though it's available on the master channel.

Reply:
Change is in follow up PR https://github.com/flutter/website/pull/6657